Assets/Scripts/PlayerController.cs

```diff
@@ -6,9 +6,12 @@ public class PlayerController : MonoBehaviour
     public LayerMask Layer;
 
     [SerializeField]
-    private const float MoveSpeed = 2f;
-    private const float JumpForce = 5.2f;
+    public const float MoveSpeed = 2.5f;
+    public const float JumpDelay = 0.2f;
+    public const float JumpForce = 5.2f;
+    public const float RunMultiplier = 1.3f;
```

Comment:
I dont think these const floats can be viewed in the inspector

Reply:
Oh. Oh well, I don't think it's too necessary to be able to edit them anyway.